test/parallel/test-buffer-alloc.js

```diff
@@ -1027,7 +1027,8 @@ assert(Buffer.allocUnsafe(1).parent instanceof ArrayBuffer);
 Buffer.poolSize = ps;
 
 // Test Buffer.copy() segfault
-assert.throws(() => Buffer.allocUnsafe(10).copy());
+assert.throws(() => Buffer.allocUnsafe(10).copy(),
+            /TypeError: argument should be a Buffer/);
```

Comment:
Lined up with arrow function. I did not match for start of string and end of string since error message that I saw were lengthy and contained much more to the string thats in validation.